### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by cakarci**

### CODE REVIEW

**Review by agriboz — COMMENTED** [review]
please change description

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 # Pull request workflow action
 
-A GitHub action that creates a workflow for your pull request including the four eyes principle for the code reviews and sending Slack notifications for the whole process
+A GitHub action that creates a workflow for your pull request for the code reviews and sending Slack notifications for the whole process
```
Comment (pcakarci) [inline README.md:7]: test review

**Pull request merged**